### PR TITLE
Add cloud links

### PR DESCRIPTION
**Pull request by Ron-Lavi**
satellite page | placement | cloud link | link text | in PR
-- | -- | -- | -- | --
/hosts | dropdown per row | https://console.redhat.com/insights/inventory/UUID | "View in Red Hat Inventory" |  
/new/hosts/UUID | main dropdown | https://console.redhat.com/insights/inventory/UUID | "View in Red Hat Inventory" |  
/new/hosts/UUID#/Overview | total-risk chart dropdown | https://console.redhat.com/insights/advisor/systems/UUID | "View in Red Hat Insights"
/new/hosts/UUID#/Insights | main tab dropdown | https://console.redhat.com/insights/advisor/systems/UUID | "View in Red Hat Insights" | V
/new/hosts/UUID#/Insights | dropdown per row | https://console.redhat.com/insights/advisor/recommendations/RECOMMENDATION_UUID | "View in Red Hat Insights" | V
/new/hosts/UUID#/Insights | dropdown per row | https://access.redhat.com/node/ARTICLE_ID | "Knowledgebase article" | V
/foreman_rh_cloud/insights_cloud | main dropdown | https://console.redhat.com/insights/advisor/systems | "View in Red Hat Insights" | V
/foreman_rh_cloud/insights_cloud | dropdown per row | https://console.redhat.com/insights/advisor/recommendations/RECOMMENDATION_UUID | "View in Red Hat Insights" | V
/foreman_rh_cloud/insights_cloud | dropdown per row | https://access.redhat.com/node/ARTICLE_ID | "Knowledgebase article" | V

### CODE REVIEW

**Comment by MariSvirik** [comment]
@Ron-Lavi UXD team has spoken and their recommendation for action links is: 
"View in Red Hat Insights" and  "Knowledgebase article" 
(Reasoning: links for knowledgebase articles are used in Insights with this exact phrasing, and "view in" should help understand that this is a specific link to get more context about this particular recommendation.)

Also, I think there should be a bit bigger space between the words and the icon.

File: webpack/InsightsHostDetailsTab/NewHostDetailsTab.js [hunk 1]
```diff
@@ -10,34 +10,55 @@ import {
...
+        <a
+          href={redHatAdvisorSystems(uuid)}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          {__('View in Red Hat Insights')}
+          {'  '}
+          <ExternalLinkAltIcon />
+        </a>
+      </DropdownItem>
```
Comment (Ron-Lavi) [inline webpack/InsightsHostDetailsTab/NewHostDetailsTab.js:58]: gnome-shell-screenshot-1YKRD1

File: webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableConstants.js [hunk 2]
```diff
@@ -28,6 +30,33 @@ export const hasPlaybookFormatter = ({ title: hasPlaybook }) => ({
...
+    );
+
+  accessRHUrl &&
+    dropdownItems.push(
+      <DropdownItem key="access-url">
+        <a href={accessRHUrl} target="_blank" rel="noopener noreferrer">
+          {__('Knowledgebase article')} <ExternalLinkAltIcon />
+        </a>
+      </DropdownItem>
+    );
```
[the remark this answers is not in the document]
Comment (Ron-Lavi) [reply webpack/InsightsCloudSync/Components/InsightsTable/InsightsTableConstants.js:53]: And the same links would work also in the table on the insights' main page.

**Comment by Ron-Lavi** [comment]
@ShimShtein I prefer to deal with the Inventory links on a follow-up as this PR got already too big and tests were failing when trying to import code from foreman 3.1

File: webpack/InsightsHostDetailsTab/NewHostDetailsTab.js (hunk 1 above)
Comment (Ron-Lavi) [reply webpack/InsightsHostDetailsTab/NewHostDetailsTab.js:58]: @MariSvirik one link goes to Satellite's Insights page and the other to the cloud, any thoughts on the two?

**Review by ShimShtein — APPROVED** [review]

**Comment by ShimShtein** [comment]
Merged, thanks @Ron-Lavi!

File: webpack/InsightsCloudSync/Components/ToolbarDropdown.js [hunk 3]
```diff
@@ -12,6 +14,17 @@ const ToolbarDropdown = ({ onRecommendationSync }) => {
     >
       {__('Sync recommendations')}
     </DropdownItem>,
+    <DropdownItem key="cloud-advisor-systems-link">
+      <a
+        href={redHatAdvisorSystems()}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        {__('View in Red Hat Insights')}
```
[the remark this answers is not in the document]
Comment (MariSvirik) [reply webpack/InsightsCloudSync/Components/ToolbarDropdown.js:23]: @Ron-Lavi Can you add a separator between those menu items? https://www.patternfly.org/v4/components/menu#with-separators. Links do not need to be blue.  
Check how PF4 treat external link icon: https://www.patternfly.org/v4/components/menu/design-guidelines#action-menu

**Pull request merged**